src/components/DevToolsPage/AccountImporter.js

```diff
@@ -79,23 +82,39 @@ class AccountImporter extends PureComponent<Props, State> {
     try {
       const { currency, xpub, isSegwit, isUnsplit } = this.state
       invariant(currency, 'no currency')
-      const derivationMode = isSegwit
-        ? isUnsplit
-          ? 'segwit_unsplit'
-          : 'segwit'
-        : isUnsplit
-          ? 'unsplit'
-          : ''
-      const rawAccount = await scanFromXPUB
-        .send({
-          seedIdentifier: 'dev_tool',
-          currencyId: currency.id,
-          xpub,
-          derivationMode,
-        })
-        .toPromise()
-      const account = decodeAccount(rawAccount)
-      this.setState({ status: 'finish', account })
+
+      if (currency.family === 'bitcoin') {
+        const derivationMode = isSegwit
+          ? isUnsplit
+            ? 'segwit_unsplit'
+            : 'segwit'
+          : isUnsplit
+            ? 'unsplit'
+            : ''
+        const rawAccount = await scanFromXPUB
+          .send({
+            seedIdentifier: 'dev_tool',
```

Comment:
we may want to add xpub here
wdyt @gre ?

Reply:
we could yeah, it really depends if you want the DevTools feature to be used with multiple different seeds at same time. To me the DevTools should never be used by random users but only devs & in a dev session (like you basically Reset App at the end of this session), in that case it's not necessary to use a seedIdentifier )